tasks/main.yml

```diff
@@ -1 +1,62 @@
-# Defaults tasks for role sync2git
+---
+# https://github.com/siteshwar/centos-sync-packages/blob/d24dfb53ce26ac3415d57961664f86484a17735b/setup.sh
+- name: "Setup system to sync packages from RHEL to CentOS"
+  hosts: all
+  tasks:
+    - name: "Install krb5-devel python3-devel python3-pip gcc and python-devel packages"
+      yum:
+        name: krb5-devel, python3-devel, python3-pip, gcc, python-devel
+        state: latest
+
+    - name: "Install koji and gitpython pip modules"
+      pip:
+        name: koji, gitpython
+
+    - name: "Install Red Hat certificate"
+      yum:
+        name: http://hdn.corp.redhat.com/rhel7-csb-stage/RPMS/noarch/redhat-internal-cert-install-0.1-15.el7.csb.noarch.rpm
+        state: present
+
+    # TODO: Is there a better way to do this in ansible ?"
+    - name: "Add yum repository"
+      shell: "yum-config-manager --add-repo http://download-ipv4.eng.brq.redhat.com/rel-eng/RCMTOOLS/rcm-tools-rhel-7-workstation.repo"
+
+    - name: "Install rhpkg"
+      yum:
+        name: rhpkg
+        state: latest
+
+    # We are mixing python2 and python3. How to handle this with ansible `pip` module ?
+    - name: "Install epel-release and python-pip"
+      yum:
+        name: epel-release, python-pip
+        state: latest
+
+    - name: "Install rpm-py-installer"
+      shell: "pip install --user rpm-py-installer"
+      become_user: centos
+
+    - name: "Install alt-src"
+      shell: "pip install --user alt-src"
+      become_user: centos
+
+    # Shall we use some other path than `/tmp/stage` for `alt-src` script ?
+    - name: "Set up /tmp/stage"
+      file:
+          name: /tmp/stage
+          state: directory
+          owner: centos
+
+    # Terrible workaround to fix dependency issues for `alt-src`
+    # https://projects.engineering.redhat.com/browse/DELIVERY-8907
+    - name: "Uninstall rpm pip"
+      shell: "pip uninstall -y rpm || :"
+      become_user: centos
+
+    - name: "Install rpm pip"
+      shell: "pip install rpm"
+
+    - name: "Copy /etc/altsrc.conf"
+      copy:
+          src: "altsrc.conf"
```

Comment:
use the template module (see first comment on altsrc.conf itself) and use variables (even empty) coming from defaults/main.yml